lib/contentful-management.ts

```diff
@@ -61,14 +65,25 @@ function createClient(
     defaults?: DefaultParams
   }
 ): PlainClientAPI
+function createClient<
+  T extends (ReadonlyArray<string> | readonly ['workflows']) &
+    { [K in keyof T]: { [P in K]: 'workflows' } }[number]
+>(
+  params: ClientOptions,
+  opts: {
+    type: 'plain'
+    alphaFeatures: T
+    defaults?: DefaultParams
+  }
+): PlainClientAPI & AlphaWorkflowExtension
 function createClient(
   params: ClientOptions,
   opts: {
     type: 'plain'
     alphaFeatures: string[]
     defaults?: DefaultParams
   }
-): AlphaPlainClientAPI
+): PlainClientAPI
```

Comment:
Are you sure about this change? Sounds like this could break a lot of typescript projects

Reply:
I just invented this alpha api on last friday (the one that wasn't released until now)